vowpalwabbit/parse_args.cc

```diff
@@ -438,6 +438,7 @@ input_options parse_source(vw& all, options_i& options)
                   "use gzip format whenever possible. If a cache file is being created, this option creates a "
                   "compressed cache file. A mixture of raw-text & compressed inputs are supported with autodetection."))
       .add(make_option("no_stdin", all.stdin_off).help("do not default to reading from stdin"))
+      .add(make_option("local", all.local).help("Do not start in daemon or active learning mode (for loaded models)"))
```

Comment:
Can you please make the option name more specific, so that it's very clear to users it only affects daemon/active

Reply:
Got it, how do you feel about `--force-local-input` or `--no-daemon`?

Reply:
I'd be okay with `--no-daemon`